.github/workflows/main-vmui.yml

```diff
@@ -0,0 +1,93 @@
+name: main-vmui
+
+on:
+  push:
+    branches:
+      - cluster
+      - master
+    paths:
+      - 'app/vmui/packages/vmui/**'
+  pull_request:
+    branches:
+      - cluster
+      - master
```

Comment:
I think it would be beneficial to run it on feature branches (PRs) as well. This way, if something is wrong, we can catch it earlier and fix it before merging into the main branch.

Reply:
Sorry, I misread the option. It means `The workflow will run on pull requests targeting cluster or master branches.`. That's good.